### PR TITLE
Added julep & OpenAI plugin

### PR DESCRIPTION
**Pull request by sawradip**

### CODE REVIEW

File: plugins/autogen/composio_autogen/autogen_toolspec.py [hunk 1]
```diff
@@ -17,11 +17,10 @@
 client = ComposioCore(
```
Comment (kaavee315) [inline plugins/autogen/composio_autogen/autogen_toolspec.py:17]: Initialize in init only?

File: plugins/openai/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+## 🦜🔗 Using Composio With LangChain
```
Comment (kaavee315) [inline plugins/openai/README.md:1]: Please change this to OpenAI.

File: plugins/openai/composio_openai/openai_toolspec.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                          entity_id: str = "default") -> list[any]:
+        output = []        
+        entity = self.client.sdk.get_entity(entity_id)
+        try:
+            if llm_response.choices:
+                for choice in llm_response.choices:
+                    if choice.message.tool_calls:
+                        for tool_call in choice.message.tool_calls:
+                            print("########")
+                            print(tool_call.function.json())
```
Comment (kaavee315) [inline plugins/openai/composio_openai/openai_toolspec.py:54]: Rm prints?

**Review by sohamganatra — COMMENTED** [review]
Bro can we use toolset specific to entity.

**Pull request closed without merging**